MapboxNavigation/CarPlayManager+Search.swift

```diff
@@ -10,6 +10,13 @@ extension CarPlayManager: CPSearchTemplateDelegate {
     public static let CarPlayGeocodedPlacemarkKey: String = "CPGecodedPlacemark"
     static var recentItems = RecentItem.loadDefaults()
     
+    /// A very coarse location manager used for focal location when searching
+    fileprivate static let coarseLocationManager: CLLocationManager = {
+        let coarseLocationManager = CLLocationManager()
+        coarseLocationManager.desiredAccuracy = kCLLocationAccuracyThreeKilometers
```

Comment:
A more fine-grained accuracy may be desirable for searching for common amenities in a dense urban area.

Reply:
Thought about this too but I reasoned that a map view is most likely already using a location manager with higher accuracy and we don't want the geocoder part of the code to be the cause draining the battery. Even if the desired accuracy is high, it's highly likely that the last received location is accurate enough, probably within 100m.